src/app/ngrx/effects/cms.effects.ts

```diff
@@ -41,7 +43,36 @@ export class CmsEffects {
     })
   );
 
+  @Effect()
+  retryActions$: Observable<Action> = this.actions$.pipe(
+    ofType(ACTIONS.ActionTypes.CMS_ACCOUNT_RETRY_ACTIONS),
+    filter((action: ACTIONS.CmsAccountRetryActionsAction) => action.payload.actions && action.payload.actions.length > 0),
+    map((action: ACTIONS.CmsAccountRetryActionsAction) => action.payload),
+    mergeMap((payload: ACTIONS.CmsAccountRetryActionsPayload) => {
+      return this.auth.token.pipe(
```

Comment:
Really feels like this shouldn't be necessary here ... if token refreshing isn't working, it's a problem [in the HalDoc lib](https://github.com/PRX/styleguide.prx.org/blob/master/src/lib/src/hal/remote/halremote.ts#L133).  Which should just seamlessly retry the last request.  I guess I'd rather try to track down the failure there, if possible.  Checking `this.auth` and parsing tokens is a bit off-topic for this app.

Reply:
Yeah, I guess I would prefer to fix it upstream in the styleguide, so it'll be fixed in Publish too.

It does seem like this does a lot of the same logic as the HalDoc, though at a higher level.  "If you get a 401, re-GET the token from ID, and retry that request once".  Maybe we should open a ticket to test this out in the styleguide?  Should be able to recreate pretty easily at a low level, by adding a char to the token or something to invalidate it.